src/har_builder.rs

```diff
@@ -0,0 +1,276 @@
+use chrono::{DateTime, Utc};
+use har::{
+    v1_2::{
```

Comment:
any reason we shouldnt use the latest module : https://docs.rs/har/0.7.1/har/v1_3/index.html

Reply:
Hey I was going of the typescript SDK and it was using 1.2